### ISSUE TITLE
Username & Password aren't autofilled on iOS

### ISSUE THREAD

**Issue by jkhsjdhjs** [thunder-app/thunder#350]
**Is your feature request related to a problem? Please describe.**
When logging into an account in the app on iOS, the keyboard offers an option to fill in username and password from stored passwords. However, when selecting the entry to fill, no data is actually inserted.

**Describe the solution you'd like**
The data should be filled in.

**Additional context**

https://github.com/thunder-app/thunder/assets/1429604/25c338e8-0f19-4ed7-8c1d-d680af032cf0

**Comment by hjiangsu** [comment]
Could you provide the password manager that you're using?

I've tested this using BitWarden on iOS and it's been working

**Comment by jkhsjdhjs** [comment]
Hmm, that's odd. I thought all password managers use the same interface, and that the problem here might be that there are 4 input fields and iOS doesn't know in which of them to fill username and password.

Anyway, I'm using Strongbox on an iPhone 8 running iOS 13.5.

**Comment by hjiangsu** [comment]
> problem here might be that there are 4 input fields and iOS doesn't know in which of them to fill username and password.

It's set up so that it chooses the username/password fields to place the information so that shouldn't be the case!

Are you by any chance logging in through email? Could you try both of them if you can? (create two entries in your password manager, one with email and the other with username)

Also if you'd like, you can try the latest nightly version and see if that maybe solves the issue https://github.com/thunder-app/thunder/releases/tag/v0.2.1-alpha.4%2B12

**Comment by hjiangsu** [comment]
I just tested it with Strongbox iPhone 13 Pro on iOS 16 and it seems to be working (v0.2.1-alpha.4+12)



https://github.com/thunder-app/thunder/assets/30667958/fa7de9be-d50f-4d3c-80df-17739ea1357e

**Comment by jkhsjdhjs** [comment]
Hmm, that's odd. I tried the nightly version and it still doesn't work, tried with both email address and username. Might be related to my older iOS version or -- since the latest Strongbox updates are only available for iOS 14+ -- the older Strongbox version (I'm using 1.58.2), but filling in passwords works fine for me with all other apps nevertheless.

I just tried filling in passwords from the iOS keyring and that actually works. Not sure where the problem resides, also haven't seen any AutoFill fixes in Strongbox since 1.58.2: https://indiecatalog.app/app/changelog/897283731

But if it works with the latest version I'm fine with closing this issue. It's not like I'm logging in and out daily anyway (well, not anymore: https://github.com/thunder-app/thunder/issues/336).
